javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,14 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  const arr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      arr.push(array[i]);
+    }
+  }
+  if (arr.length > 0) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You don't need this `if` statement at all. You can just `return true` on line 6, and then outside the loop, you can `return false`. Since `return` causes us to exit out of a function, we wouldn't have to worry about any other code being ran. Pushing to this `arr` and checking the length is just more work that isn't needed!